src/globus_sdk/services/compute/client.py

```diff
@@ -101,18 +101,22 @@ def get_endpoint_status(self, endpoint_id: UUIDLike) -> GlobusHTTPResponse:
         """  # noqa: E501
         return self.get(f"/v2/endpoints/{endpoint_id}/status")
 
-    def get_endpoints(self) -> GlobusHTTPResponse:
+    def get_endpoints(self, role: str | None = None) -> GlobusHTTPResponse:
         """Get a list of registered endpoints associated with the authenticated user.
 
-        .. tab-set::
+                :param role: Role of the user in relation to endpoints. (e.g.: owner, any)
```

Comment:
This block looks like it was indented quite a bit. I'm not convinced this needs to be indented further.

```suggestion
        :param role: Role of the user in relation to endpoints. (e.g.: owner, any)